tasks/input_sql_file.yml

```diff
@@ -0,0 +1,35 @@
+- name: Verify if postgresql_password is set
+  assert:
+    that: postgresql_password is defined
+    fail_msg: >-
+      You must define postgresql_password variable to be able accessing
+      the database.
+
+- name: Copy SQL file on the host
+  copy:
+    src: "{{ postgresql_input_file }}"
+    dest: "/tmp/input.sql"
+    owner: postgres
+    group: postgres
+    mode: '0666'
+  tags:
+    - dir_not_exist
+
+- name: Run sql script from {{ postgresql_input_file }}
+  become: yes
+  become_user: postgres
+  environment:
+    PGPASSWORD: "{{ postgresql_password }}"
+  command:
+    cmd: psql -U postgres -f "/tmp/input.sql"
+  register: psql_output
```

Comment:
For variables used internally by the role - not intended to be set by users - use naming format like this: `__postgresql_varname`